gunicorn/sock.py

```diff
@@ -12,6 +12,7 @@
 
 from gunicorn import util
 from gunicorn.six import string_types
+from gunicorn.socketfromfd import fromfd
```

Comment:
I know you've followed the current format, but I wonder if we should rename the module to `_socketfromfd`. We are going to remove `argparse_compat.py`, `six.py` and `selectors.py` in 20 and `socketfromfd` will probably be our only vendored dependency (and if we decide not to support Python 3.3 we can remove `socketfromfd` as well)

Reply:
@berkerpeksag Why can we remove `socketfromfd` if Py 3.3 is dropped? `socketfromfd` provides functionality which is not available up to Py 3.7 (namely deducing the socket family/type from the fd)

Reply:
Good point, I only checked when `fromfd` was added to the stdlib.

Reply:
@apollo13 could you add a comment in `gunicorn/socketfromfd.py` to explain that socketfromfd is needed for Python versions between 3.3 and 3.6 so we won't accidentally remove it in the future.